src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int indexBall = new Random().nextInt(100);
```

Comment:
The magic number 100 should be replaced with a named constant to improve code readability and maintainability.

Reply:
Fixed